Test/FMI3/parser_test_xmls/model_structure/invalid/incorrect_order/modelDescription.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<fmiModelDescription
+  fmiVersion="3.0"
+  modelName="myModelName"
+  instantiationToken="myInstToken">
+
+<ModelExchange
+  modelIdentifier="myModelIdentifier" />
+
+<ModelVariables>
+    <Float64 name="state_var" valueReference="0"/>
+    <Float64 name="der(state_var)" valueReference="1" causality="output" variability="continuous" initial="approx" derivative="1" start="1.0"/>
+    <Float64 name="output_var" valueReference="2"/>
```

Comment:
causality=output should be on output_var

Reply:
Fixed, also with error check on this now.